### PR TITLE
Lilly/Megan Pine C16

### PR DESCRIPTION
**Pull request by waterlilly169**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+    Planet(6, "Jupiter", "Sixth planet in our solar system", "God of sky and thunder"),
+    Planet(7, "Uranus", "Seventh planet in our solar system", "God of sky"),
+    Planet(8, "Neptune", "Eighth planet, in our solar system", "God of the oceans"),
+    Planet(9, "Not Pluto", "Not quite a planet", "God of disillusuinment")
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix ="/planets")
+
+@planets_bp.route("", methods=["GET"])
+def get_all_planets():
```
Comment (jbieniosek) [inline app/routes.py:26]: Great work!

File: app/routes.py [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+def get_single_planet(planet_id):
+    planet_id = int(planet_id)
+    for each_planet in planet_instances:
+        if each_planet.id == planet_id:
+            return {
+                "id": each_planet.id,
+                "name": each_planet.name,
+                "description": each_planet.description,
+                "mythology": each_planet.mythology,
+            }
```
Comment (jbieniosek) [inline app/routes.py:48]: I recommend adding ‘jsonify’ to the return even in the case where the Flask default behavior is doing the same thing. Using ‘jsonify’ adds a layer of error protection and also makes it clear that json is the intended behavior.
```suggestion
            return jsonify({
                "id": each_planet.id,
                "name": each_planet.name,
                "description": each_planet.description,
                "mythology": each_planet.mythology,
            })
```

File: app/routes.py [hunk 3]
```diff
@@ -1,2 +1,48 @@
 from flask import Blueprint
+from flask import Blueprint, jsonify
 
+class Planet():
+    def __init__(self, id, name, description, mythology):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.mythology = mythology
```
Comment (jbieniosek) [inline app/routes.py:9]: 😄

File: tests/test_routes.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+def test_get_one_planet_with_record(client, create_two_planets):
+    #Act
+    response = client.get("/planets/1")
```
Comment (CheezItMan) [inline tests/test_routes.py:3]: This is a really in the weeds comment, but I suggest avoiding hard-coded numbers like this.  Instead either find the ID from the DB with the Model methods, or make it a constant you can lookup.

This is a very picky/opinionated comment.

**Review by CheezItMan — COMMENTED** [review]
Really nicely done Lilly & Megan.  I really like the tests you wrote.  Well done.

**Pull request closed without merging**